protoactor/props.py

```diff
@@ -0,0 +1,102 @@
+from messages import Started
+from context import get_default_receive
+
+def get_default_spawner(name, props, parent):
+    ctx = Context(props.producer, props.supervisor_strategy, props.middleware_chain, parent)
+    mailbox = props.mailbox_producer
+    dispatcher = props.dispatcher
+    reff = LocalProcess(mailbox)
+    pr = ProcessRegistry().add(name, reff)
+    ctx.self = pid
+
+    mailbox.register_handlers(ctx, dispatcher)
+    mailbox.post_system_message(Started())
+    mailbox.start()
+
+    return pid
+
+def middleware_chain_concat(middleware):
+    rv_list = list(reverse(self.__middleware))

```

Comment:
I think this should be changed from `self.__middleware` to `middleware` or am I missing something?